test/I.js

```diff
@@ -18,4 +20,12 @@ describe('I', function() {
        {'@@type': 'my-package/Foreign'});
   });
 
+  it('is idempotent', function() {
+    ok(properties.idempotent(S.I), true);
```

Comment:
Good catch! Yeah the origin of `ok(..., true)` is a misdiagnosed error message (I thought it was about `eq`). Will change